content/developer/reference/backend/orm.rst

```diff
@@ -889,7 +889,37 @@ Search/Read
 
 .. automethod:: Model._read_group
 
-.. automethod:: Model.read_group
+.. method:: Model.web_formatted_read_group(self, domain, groupby=(), aggregates=(), having=(), offset=0, limit=None, order=None)
```

Comment:
Why not use `automethod` and avoid duplication of documentation?

Reply:
`web_formatted_read_group` is defined on web. I tried to make it work without success. I will try again.

Reply:
Maybe it should not be documented in this page. Now it became a function to be used only for front-end development and is no longer part of the ORM.